cmd/bnsd/app/app.go

```diff
@@ -101,6 +103,7 @@ func QueryRouter() weave.QueryRouter {
 		orm.RegisterQuery,
 		currency.RegisterQuery,
 		distribution.RegisterQuery,
+		antiSpamQuery.RegisterQuery,
```

Comment:
Interesting.. I would assume that antiSpamQuery could be a RegisterQuery function already

Reply:
After reading the code, very nice choice :+1: